tck/src/main/java/ee/jakarta/tck/data/web/transaction/PersistenceTests.java

```diff
@@ -0,0 +1,109 @@
+import ee.jakarta.tck.data.framework.junit.anno.Transaction;
+import ee.jakarta.tck.data.framework.junit.anno.Web;
+import ee.jakarta.tck.data.standalone.persistence.Catalog;
+import ee.jakarta.tck.data.standalone.persistence.Product;
+import ee.jakarta.tck.data.standalone.persistence.Product.Department;
+//import jakarta.annotation.Resource;
+import jakarta.inject.Inject;
+import jakarta.transaction.UserTransaction;
+
+/*
+ * TODO would it be possible to use Transactions in a standalone environment?
+ * If so how would we verify that transactions are available?
+ */
+@Web
+@Persistence
+@Transaction
+public class PersistenceTests {
+
+    @Deployment
+    public static WebArchive createDeployment() {
+        return ShrinkWrap.create(WebArchive.class).addClasses(Catalog.class, Product.class);
+    }
+
+    @Inject
+    UserTransaction tran;
+
+    @Inject
+    Catalog catalog;
+
+    @Assertion(id = "133", strategy = "Verify that data is persisted to repository when a transaction is committed")
+    public void testCommit() throws Exception {
+        catalog.deleteByProductNumLike("TEST-PROD-%");
+
+        List<Product> yarns;
+
+        tran.begin();
+
+        catalog.save(Product.of("wool-yarn", 7.99, "TEST-PROD-100", Department.CRAFTS));
+        catalog.save(Product.of("cotton-yarn", 5.99, "TEST-PROD-101", Department.CRAFTS));
+        catalog.save(Product.of("acylic-yarn", 3.99, "TEST-PROD-102", Department.CRAFTS));
+
+        yarns = catalog.findByProductNumLike("TEST-PROD-10_");
+        assertTrue(yarns.isEmpty(), 
+                "Save should not have persisted data prior to transaction commit, but instead found " + yarns.size() + " results");
+
+        tran.commit();
+
+        yarns = catalog.findByProductNumLike("TEST-PROD-10_");
+        assertFalse(yarns.isEmpty(), "Commit should have persisted data");
+        assertEquals(3, yarns.size());
+
+        Product woolYarn = yarns.stream().filter(p -> p.getProductNum() == "TEST-PROD-100").findFirst().orElse(null);
+        assertNotNull(woolYarn, "Result set did not contain the expected product");
+        assertEquals("TEST-PROD-100", woolYarn.getProductNum());
+    }
+
+    @Assertion(id = "133", strategy = "Verify that data is not persisted to repository when a transaction is rolledback")
+    public void testRollback() throws Exception {
+        catalog.deleteByProductNumLike("TEST-PROD-%");
+
+        List<Product> paints;
+
+        tran.begin();
+
+        catalog.save(Product.of("oil-paint", 12.55, "TEST-PROD-110", Department.CRAFTS));
+        catalog.save(Product.of("acrylic-paint", 6.55, "TEST-PROD-111", Department.CRAFTS));
+        catalog.save(Product.of("watercolor", 5.55, "TEST-PROD-112", Department.CRAFTS));
+
+        paints = catalog.findByProductNumLike("TEST-PROD-11_");
+        assertTrue(paints.isEmpty(), 
+                "Save should not have persisted data prior to transaction commit, but instead found " + paints.size() + " results");
```

Comment:
Same reason as prior comment:
```suggestion
        assertEquals(3, paints.size()); 
```